src/ResourceManager/Billing/Commands.Billing.Test/SessionRecords/Microsoft.Azure.Commands.Billing.Test.ScenarioTests.BillingPeriodsTests/TestGetBillingPeriodWithNames.json

```diff
@@ -1,8 +1,8 @@
 {
   "Entries": [
     {
-      "RequestUri": "/subscriptions/0347d63a-421a-43a7-836a-5f389c79cd07/providers/Microsoft.Billing/billingPeriods/201705-1?api-version=2017-04-24-preview",
```

Comment:
same question on all of these updated recordings - did you re-record, or did you edit the recording?

Reply:
Got recording to work. All have been re-recorded now.